### PR TITLE
feat: datalake-interface-exporter

### PR DESCRIPTION
**Pull request by micdes-pagopa**

### CODE REVIEW

File: kubernetes/configs/es1-dev/versions.sh [hunk 1]
```diff
@@ -1,52 +1,53 @@
...
-TENANT_OUTBOUND_WRITER_IMAGE_VERSION="2.x-latest"
-API_GATEWAY_IMAGE_VERSION="2.x-latest"
+PURPOSE_OUTBOUND_WRITER_IMAGE_VERSION="develop"
+PURPOSE_READMODEL_WRITER_IMAGE_VERSION="develop"
+PURPOSE_PROCESS_IMAGE_VERSION="develop"
+TENANT_PROCESS_IMAGE_VERSION="develop"
+TENANT_READMODEL_WRITER_IMAGE_VERSION="develop"
+TENANT_OUTBOUND_WRITER_IMAGE_VERSION="develop"
+API_GATEWAY_IMAGE_VERSION="develop"
 BACKEND_FOR_FRONTEND_IMAGE_VERSION="2.x-latest"
```
Comment (beetlecrunch) [inline kubernetes/configs/es1-dev/versions.sh]:
```suggestion
BACKEND_FOR_FRONTEND_IMAGE_VERSION="develop"
```

File: kubernetes/configs/es1-test/versions.sh [hunk 2]
```diff
@@ -15,6 +15,7 @@ CATALOG_OUTBOUND_WRITER_IMAGE_VERSION="2.0.19-RC1"
 CATALOG_READMODEL_WRITER_IMAGE_VERSION="2.0.19-RC1"
 CATALOG_PROCESS_IMAGE_VERSION="2.0.19-RC1"
 COMPUTE_AGREEMENTS_CONSUMER_IMAGE_VERSION="2.0.19-RC1"
+DATALAKE_INTERFACE_EXPORTER_IMAGE_VERSION="2.0.20-RC1"
```
Comment (beetlecrunch) [inline kubernetes/configs/es1-test/versions.sh:18]: 👍

**Review by beetlecrunch — DISMISSED** [review]

File: kubernetes/configs/es1-dev/versions.sh (hunk 1 above)
Comment (micdes-pagopa) [reply kubernetes/configs/es1-dev/versions.sh]: Is it fixed now? It was missing some delegation envs

Comment (beetlecrunch) [reply kubernetes/configs/es1-dev/versions.sh]: Not yet, but I gonna fixit after this PR, ok?

**Review by beetlecrunch — DISMISSED** [review]

**Review by beetlecrunch — DISMISSED** [review]

**Review by beetlecrunch — DISMISSED** [review]

**Pull request merged**